homeassistant/components/history/__init__.py

```diff
@@ -46,6 +33,8 @@
     {
         DOMAIN: vol.All(
             cv.deprecated(CONF_ORDER),
+            cv.deprecated(CONF_INCLUDE),
```

Comment:
Shouldn't we just put `cv.deprecated` on `DOMAIN` ?

Reply:
Yes. That makes a lot more sense.

Reply:
0bb16cc1c84f1def923cc39ad39fbe35ccf6de27